### PR TITLE
Pe 2178

### PR DESCRIPTION
**Pull request by agsuy**
- Fixes Flutter version lefthook script.
- Adds VSCode workspace with .yaml configuration.
- Updates the Readme.
- Updates .gitignore to ignore.

### CODE REVIEW

File: ardrive-web.code-workspace [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+    ],
+    "settings": {
+        "dart.additionalAnalyzerFileExtensions": [
+            "moor"
```
Comment (karlprieb) [inline ardrive-web.code-workspace:9]: we changed to `drift`

File: README.md [hunk 2]
```diff
@@ -30,6 +30,16 @@ Then, generate the package imports with:
...
 ```
+Install script runner to have access to pubspec scripts
+```
+pub global activate script_runner
+```
+
+In order to run them:
+
+```
+scr my-script ...args
```
Comment (karlprieb) [inline README.md:41]: please provide some examples using real scripts like `scr setup`

File: ardrive-web.code-workspace (hunk 1 above)
Comment (agsuy) [reply ardrive-web.code-workspace:9]: I blve its the same DB and hence the same analyzer. Just a "rebrand"

File: README.md (hunk 2 above)
Comment (agsuy) [reply README.md]: Added

File: ardrive-web.code-workspace (hunk 1 above)
Comment (karlprieb) [reply ardrive-web.code-workspace:9]: Here we can see that the name of the file extension change to `drift` https://drift.simonbinder.eu/docs/using-sql/sql_ide/

File: dart_test.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# This is a browser-only package, so test on chrome by default.
+platforms:
```
Comment (thiagocarvalhodev) [inline dart_test.yaml:2]: Is that intentional? I'm asking because we don't fully understand how to proceed with that for now.

**Review by thiagocarvalhodev — APPROVED** [review]
LGMT! Left just one question.

File: dart_test.yaml (hunk 3 above)
Comment (agsuy) [reply dart_test.yaml:2]: Its not. :sob:  

I ll go and cry on a corner for a few minutes, then remove that file.

File: ardrive-web.code-workspace (hunk 1 above)
Comment (agsuy) [reply ardrive-web.code-workspace:9]: Fixed.

Comment (agsuy) [reply ardrive-web.code-workspace:9]: :facepalm:

**Review by karlprieb — APPROVED** [review]

**Pull request merged**